packages/core/addon/adapters/base-json-adapter.js

```diff
@@ -69,7 +71,7 @@ export default DS.JSONAPIAdapter.extend({
    * @return {Object} errors payload
    */
   normalizeErrorResponse(status, headers, payload) {
-    let detail = Ember.get(payload, 'errors');
+    let detail = get(payload, 'errors');
```

Comment:
`payload.errors` ?

Reply:
In the general case `get(something, 'prop')` -> `something.prop` might cause errors if `something` is not an object.

Reply:
There's already enough changes in this pr and I see no need to mess with the `get`s at this point.